pkg/bot/interactive/help.go

```diff
@@ -168,6 +168,9 @@ func (h *HelpMessage) kubectlSections() []Section {
 				},
 			},
 			{
+				Base: Base{
+					Description: "Build kubectl commands interactively",
+				},
```

Comment:
Are you sure this change is needed? We have "Interactive kubectl - no typing!" header already:

![image](https://user-images.githubusercontent.com/7155799/203927663-45ed8fcb-ff50-4393-bb26-2ebc592fa6d4.png)